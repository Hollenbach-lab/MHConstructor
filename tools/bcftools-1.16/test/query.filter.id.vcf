##fileformat=VCFv4.1
##reference=file:///seq/references/1000Genomes-NCBI37.fasta
##contig=<ID=1>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	123	rs123	A	.	.	.	.
1	124	ss124;abc	A	.	.	.	.
1	125	.	A	.	.	.	.
