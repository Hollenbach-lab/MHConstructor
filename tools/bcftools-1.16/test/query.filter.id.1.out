rs123
ss124;abc
